components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -251,6 +251,38 @@ paths:
             schema:
               $ref: '#/components/schemas/PatchRequest'
         required: true
+
+  /userstores/{userstore-domain-id}/attribute-mappings:
+    patch:
+      tags:
+        - User Store
+      summary: Patch the secondary user store attribute mappings by it's domain id.
```

Comment:
Let's say it's an update operation instead of a patch.

Reply:
Addressed with commit c731518fd0ea44d2d06c025d044e61114470